### PR TITLE
Add /tools/vs-code

### PR DESCRIPTION
**Pull request by kwalrath**
And clean up some /tools page links.

### CODE REVIEW

File: src/tools/vs-code.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+title: Visual Studio Code
+description: You can use VS Code to develop Dart apps.
+toc: false
+---
+
+With the Dart plugin, you can use [Visual Studio Code (VS Code)][vs-code]
+to develop Dart apps.
+Here's where you can find more information:
+
+* [Set up an editor][setup] on the Flutter site has
```
Comment (mit-mit) [inline src/tools/vs-code.md]: Maybe clarify that this also works for Dart command-line and server apps?

Comment (kwalrath) [reply src/tools/vs-code.md]: You mean the Flutter instructions, or VS Code in general? I thought it worked for web apps, too. (Obviously, I haven't tried it.)

Comment (mit-mit) [reply src/tools/vs-code.md]: VS Code works great for all kinds of Dart apps: command-line, server, web, and Flutter. But the current phrasing and link to Flutter can be mis-read to suggest it's for Flutter only.

**Comment by kwalrath** [comment]
ptal

**Pull request merged**